tests/test_lint_recipe.py

```diff
@@ -824,28 +824,30 @@ def test_maintainer_exists(self):
         expected_message = 'Recipe maintainer "isuruf" does not exist'
         self.assertNotIn(expected_message, lints)
 
-        expected_message = "Feedstock with the same name exists in conda-forge"
+        expected_message = (
+            "Feedstock with the same name exists in conda-forge."
+        )
         # Check that feedstock exists if staged_recipes
-        lints = linter.lintify(
+        lints, _ = linter.lintify(
```

Comment:
I'm a little confused about this, was the test broken beforehand? Does line 821 also need to be fixed?

Reply:
Yes, they weren't tested before. I fixed the testing in c0d5c627da8e7566559d1a60bc273bd90abb1e90